docs/conf.py

```diff
@@ -0,0 +1,59 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+import os
+import sys
+
+sys.path.insert(0, os.path.abspath('../appium'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'Python client 1.0 beta'
```

Comment:
why beta?

Reply:
Ah, I thought we will release 1.0beta at first, so just for it.
(It's not matched with https://github.com/appium/python-client/pull/526/files/6235b6a8d0339a92fa2fd2ff2e79d7c375e5143c#diff-2eeaed663bd0d25b7e608891384b7298R40 )

If any ideas, please let me know. > @mykola-mokhnach , @KazuCocoa 
I'll remove beta for this PR.

Reply:
I vote for removing beta. The stuff has been working for quite a while and is stable enough